src/components/shared/BracketArrow.js

```diff
@@ -1,25 +1,31 @@
 // @flow
 
-import { DOM as dom } from "react";
+import { DOM as dom, PropTypes, Component } from "react";
 import classNames from "classnames";
 
 import "./BracketArrow.css";
 
-const BracketArrow = (
-  orientation: string,
-  left: number,
-  top: number,
-  bottom: number
-) => {
-  return dom.div(
-    {
-      className: classNames("bracket-arrow", orientation || "up"),
-      style: { left, top, bottom }
-    },
-    ""
-  );
-};
+class BracketArrow extends Component {
+  render() {
+    const { orientation, left, top, bottom } = this.props;
+
+    return dom.div(
+      {
+        className: classNames("bracket-arrow", orientation || "up"),
+        style: { left, top, bottom }
+      },
+      ""
+    );
+  }
+}
 
 BracketArrow.displayName = "BracketArrow";
 
+BracketArrow.propTypes = {
+  orientation: PropTypes.string,
+  left: PropTypes.number,
+  top: PropTypes.number,
+  bottom: PropTypes.number
+};
```

Comment:
We're now using props, which is flow based.

Checkout https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/Footer.js#L25-L35